csrc/scheduler/registry.cpp

```diff
@@ -1686,14 +1714,6 @@ class PointWiseScheduler : public SchedulerEntry {
       }
     }
 
-    auto reduction_ops = ir_utils::getReductionOps(fusion);
```

Comment:
Where's the corresponding check of the eligibility of using the pointwise scheduler with reductions?

Reply:
WDYM?